From d5a4787c00900a0a637507c978b3fec13edaf34b Mon Sep 17 00:00:00 2001
From: Simon Fels <simon.fels@canonical.com>
Date: Mon, 1 Aug 2016 12:08:13 +0200
Subject: [PATCH 4/4] data/systemd: don't fail to start if /etc/environment
 does not exist

On some systems /etc/environment is not available by default and
should be prevent snapd from starting.
---
 data/systemd/snapd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/systemd/snapd.service.in b/data/systemd/snapd.service.in
index 009e62e..35f083d 100644
--- a/data/systemd/snapd.service.in
+++ b/data/systemd/snapd.service.in
@@ -4,7 +4,6 @@ Requires=snapd.socket
 
 [Service]
 ExecStart=@libexecdir@/snapd/snapd
-EnvironmentFile=@SNAPD_ENVIRONMENT_FILE@
 Restart=always
 
 [Install]
-- 
2.7.4

